src/handbook/company/security/human-resources.md

```diff
@@ -0,0 +1,120 @@
+of the hiring process. Required skills and competencies shall be listed in job
+descriptions and requisitions, and/or aligned with the responsibilities outlined
+in the Information Security Roles and Responsibilities Policy. Competency
+evaluations may include reference checks, education and certification verifications,
+technical testing, and interviews.
+
+All FlowForge employees will undergo an annual performance review which will
+include an assessment of job performance, competence in the role, adherence to
+company policies and code of conduct, and achievement of role-specific objectives.
+
+
+### Terms & Conditions of Employment
+
+Company policies and information security roles and responsibilities shall be
+communicated to employees and third-parties at the time of hire or engagement,
+and employees and contractors are required to formally acknowledge their
+understanding and acceptance of their security responsibilities.
+
+Employees and relevant third-parties shall follow all FlowForge information
+security policies.
+
+### Management Responsibilities
+
+Management shall be responsible for ensuring that information security policies
+and procedures are reviewed annually, distributed and available, and that
+employees and contractors abide by those policies and procedures for the duration
+of their employment or engagement.
+
+Annual policy review shall include a review of any linked or referenced procedures,
+standards or guidelines.
+
+Management shall ensure that information security responsibilities are
+communicated to individuals, through written job descriptions, policies or some
+other documented method which is accurately updated and maintained.
+
+Compliance with information security policies and procedures and fulfillment of
+information security responsibilities shall be evaluated as part of the performance
+review process wherever applicable.
+
+Management shall consider excessive pressures, and opportunities for fraud when
+establishing incentives and segregating roles, responsibilities, and authorities.
+
+
+### Information Security Awareness, Education & Training
+
+All FlowForge employees and third-parties with administrative or privileged
+technical access to FlowForge production systems and networks shall complete
+security awareness training at the time of hire and annually thereafter.
+
+Management shall monitor training completion and shall take appropriate steps to
+ensure compliance with this policy. Employees and contractors shall be aware of
+relevant information security and data privacy policies and procedures. The
+company shall ensure that personnel receive security and data privacy training
+appropriate to their role and data handling responsibilities.
+
+In order to maintain a robust level of security awareness, the company will
+provide security-related updates and communications to company personnel on an
+on-going basis through multiple communication channels as needed.
+
+### Termination Process
```

Comment:
@knolleary There's already a section around this in the handbook. I'd rather not duplicate all this. Please link to our termination process instead of duplicating